slack.py

```diff
@@ -4,24 +4,45 @@
 url = "https://hooks.slack.com/services/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"
```

Comment:
please note that I now get 404 when trying to send the message to the webhook, so we might not want to play with it anymore